libraries/lib-ipc/internal/BufferedIPCChannel.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <vector>
+
+#include "ipc-types.h"
+#include "IPCChannel.h"
+#include "spinlock.h"
+
+class IPCChannelStatusCallback;
+
+/**
+ * \brief Implementation of IPCChannel that uses intermediate
+ * buffer for data exchange between client and server.
+ */
+class BufferedIPCChannel final : public IPCChannel
+{
+   spinlock mSync;
+
+   IPCChannelStatusCallback* mCallback;
+
+   SOCKET mSocket;
```

Comment:
Can this member be const?
It's always good to add const when you can to help the reader understand what parts never vary after construction.